src/components/forms/Radio/Radio.tsx

```diff
@@ -20,14 +21,18 @@ export const Radio = ({
   className,
   label,
   inputRef,
+  tile,
   ...inputProps
 }: RadioProps & JSX.IntrinsicElements['input']): React.ReactElement => {
   const classes = classnames('usa-radio', className)
+  const radioClass = classnames('usa-radio__input', {
```

Comment:
Super minor, but can we make this `radioClasses`? I think its just a general convention I've noticed, and it makes sense since there could me multiple CSS classes applied.